src/lib/CalibrationVerification.cpp

```diff
@@ -601,8 +601,8 @@ void CalibrationVerification::SaveCameraVisuals() {
       LoadLookupTree();
 
       // load image from bag
-      std::shared_ptr<cv::Mat> current_image = GetImageFromBag(
-          params_->camera_params[measurement->camera_id]->topic);
+      std::shared_ptr<cv::Mat> current_image = std::make_shared<cv::Mat>();
+      *current_image = measurement->img;
```

Comment:
I don't think this even needs to be a pointer, can you change it to just a cv::Mat?

Reply:
Also, can we remove the GetImageFromBag function now?

Reply:
> Also, can we remove the GetImageFromBag function now?

Removed